src/Radio/RadioGroup.js

```diff
@@ -15,131 +15,90 @@
+  /** Data used to contruct [Radio](../radio), see [example](#data) */
   data?: Array<{ label: string | React$Element<*>, value: string }>,
   /**
-   * Value of the selected Radio; primarily for use with uncontrolled
-   * components
+   * Value of the selected [Radio](../radio); primarily for use with
+   * uncontrolled components.
    */
   defaultChecked?: string,
-  /** Display the choices inline horizontally rather than stacked vertically */
+  /** Display the choices inline horizontally rather than stacked vertically. */
   inline?: boolean,
-  /**
-   * The name of the radio group; used to uniquely define a group of radio
-   * buttons
-   */
+  /** The name of the group */
   name: string,
-  /** Function called when a radio button is selected */
+  /** Function called when a choice is selected */
   onChange?: (event: SyntheticEvent<>) => void,
   /** Props to be applied directly to the root element */
-  rootProps?: Object,
-  /** @Private Available sizes */
-  size?: 'small' | 'medium' | 'large' | 'jumbo'
+  rootProps?: Object
 };
 
-export const componentTheme = (baseTheme: Object) => ({
-  RadioGroup_marginHorizontal_inline: baseTheme.space_inline_xl,
-  RadioGroup_marginVertical: baseTheme.space_stack_md,
-  RadioGroup_marginVertical_jumbo: baseTheme.space_stack_lg,
-
-  ...baseTheme
-});
-
-const styles = {
-  root: ({ inline, size, theme: baseTheme }) => {
-    const theme = componentTheme(baseTheme);
-
-    return {
-      display: 'flex',
-      flexDirection: inline ? 'row' : 'column',
-
-      '& > *:not(:last-child)': {
-        marginBottom: inline
-          ? 0
-          : size === 'jumbo'
-            ? theme.RadioGroup_marginVertical_jumbo
-            : theme.RadioGroup_marginVertical,
-        marginRight: inline
-          ? theme.RadioGroup_marginHorizontal_inline
-          : undefined
-      }
-    };
-  }
+export const componentTheme = (baseTheme: Object) => {
+  return {
+    ...mapComponentThemes(
+      {
+        name: 'ChoiceGroup',
+        theme: choiceGroupComponentTheme(baseTheme)
+      },
+      {
+        name: 'RadioGroup',
+        theme: {}
+      },
+      baseTheme
+    )
+  };
 };
 
-const Root = createStyledComponent('div', styles.root, {
-  displayName: 'RadioGroup',
-  includeStyleReset: true
+const Root = createThemedComponent(ChoiceGroup, ({ theme: baseTheme }) => {
+  return {
+    ...mapComponentThemes(
+      {
+        name: 'RadioGroup',
+        theme: componentTheme(baseTheme)
+      },
+      {
+        name: 'ChoiceGroup',
+        theme: {}
+      },
+      baseTheme
+    )
+  };
 });
 
 /**
- * RadioGroup allows users to select a single option from a set and provides a
- * simpler API than working with [Radios](../radio) directly.
+ * RadioGroup allows authors to construct a group of [Radios](../radio)
+ * and provides a simpler API than working with Radio directly.
+ *
+ * RadioGroup allows users to select a single option from a set.
```

Comment:
We used "list" rather than "set" in CheckboxGroup.

Reply:
Updated.